src/stackwalk.c

```diff
@@ -37,7 +57,7 @@ static int jl_unw_step(bt_cursor_t *cursor, uintptr_t *ip, uintptr_t *sp, uintpt
 // entries.  If `sp != NULL`, the stack pointer corresponding `bt_data[i]` is
 // stored in `sp[i]`.
 //
-// Flag `add_interp_frames==1` should be set to record an extended backtrace
+// `pgcstack` should be given if you want to record extended backtrace
```

Comment:
```suggestion
// `ppgcstack` should be given if you want to record extended backtrace
```

Reply:
It's `pgcstack` that must be given, but I guess it make more sense to encode this as:
```suggestion
// `*ppgcstack` should be given if you want to record extended backtrace
```

Reply:
Yes that's much better.